CPP11.md

```diff
@@ -933,6 +934,27 @@ C++11 introduces a memory model for C++, which means library support for threadi
 
 See the sections on: [std::thread](#stdthread)
 
+### Random functions
+According to CERT, the rand function doesn't have "good statistical properties" and can be predictable, which makes programs less secure (CERT Guideline MSC30-CPP), C++11 has a new and more secure library of tools for generating random numbers that can produce non-deterministic numbers for simulations and security cases, where predictability is not convenient. These new tools are found in the <random> header of the C++ standard library.
+An engine implements a random number generation algorithm that produces pseudo-random numbers. A distribution controls the range of values produced by an engine, the types of values (for example int, double, etc), and the statistical properties of the values.
+`std::default_random_engine` it´s the motor for default
+`std::uniform_int_distribution`evenly distributes pseudo-random integers across a specified range of values, the default range is 0 to the maximum value of an int on your platform
```

Comment:
The markdown doesn't render nicely for this line and the line above it. I think it would be useful to make them bulleted. Also forgot a space before the word "evenly".